### PR TITLE
75119 persist po info on supplier area tag

### PR DESCRIPTION
**Pull request by janid1967**

### CODE REVIEW

File: src/Equinor.Procosys.Preservation.Command/TagCommands/CreateAreaTag/CreateAreaTagCommandHandler.cs [hunk 1]
```diff
@@ -114,6 +114,19 @@ private async Task<Tag> CreateAreaTagAsync(CreateAreaTagCommand request)
                 requirements.Add(new TagRequirement(_plantProvider.Plant, requirement.IntervalWeeks, reqDef));
             }
 
+            string purchaseOrderNo = null;
+            string calloff = null;
+
+            if (request.TagType == TagType.PoArea)
+            {
+                var poParts = request.PurchaseOrderCalloffCode.Split('/');
```
Comment (hbjorgo) [inline src/Equinor.Procosys.Preservation.Command/TagCommands/CreateAreaTag/CreateAreaTagCommandHandler.cs:122]: Do we need a check to see if the split results in an array with length > 0?

Comment (janid1967) [reply src/Equinor.Procosys.Preservation.Command/TagCommands/CreateAreaTag/CreateAreaTagCommandHandler.cs:122]: Yes, should be highlighted when reading unit tests

Comment (janid1967) [reply src/Equinor.Procosys.Preservation.Command/TagCommands/CreateAreaTag/CreateAreaTagCommandHandler.cs:122]: In selector in UI, either PO or PO/CO can be chosen

Comment (janid1967) [reply src/Equinor.Procosys.Preservation.Command/TagCommands/CreateAreaTag/CreateAreaTagCommandHandler.cs:122]: What I should do, is to throw excdeption if PurchaseOrderCalloffCode is null or empty

**Pull request merged**